src/App.vue

```diff
@@ -366,6 +367,94 @@ export default {
         return data.pendingInvitations
       }
     })
+
```

Comment:
This is the wrong location for this logic - it should most likely be in its own file in the `plugins` directory and initialized in `main.js` after checking that we're in a cloud environment (the node_env shouldn't matter)